docs/api/api/configuration.rng

```diff
@@ -1,26 +1,29 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<grammar ns="http://buildservice.org/api" xmlns="http://relaxng.org/ns/structure/1.0" datatypeLibrary="http://www.w3.org/2001/XMLSchema-datatypes">
-
-  <include href="obs.rng" />
-
+<grammar xmlns="http://relaxng.org/ns/structure/1.0" xmlns:a="http://www.example.com/annotation" ns="http://buildservice.org/api" datatypeLibrary="http://www.w3.org/2001/XMLSchema-datatypes">
```

Comment:
example.com?

Reply:
uh, looks like a copy-pasta fail... Should be fixed now.